libraries/botbuilder/package.json

```diff
@@ -49,14 +49,16 @@
     "node-mocks-http": "^1.16.0"
   },
   "scripts": {
-    "build": "tsc -b",
+    "build": "npm-run-all clean build:tsc build:tsup",
+    "build:tsc": "tsc -b",
+    "build:tsup": "tsup src/ --format cjs,esm --sourcemap --outDir lib",
```

Comment:
I think we could use cjs format only, since we dont have the package.json configured to support esm yet